test/func/mocha.opts

```diff
@@ -0,0 +1,3 @@
+test/func/spec/**/*.spec.js
+--recursive
+--timeout 50000
```

Comment:
Hmmm... For now, can we just `this.timeout()` in a `before` and maybe prime the server with a single request? (Can be a separate PR, but I'd love to not have CI / our tests at risk of huge wait times in the case of bad tests...)

There are other rearchitecting ways to separately account for the webpack build which should not make all the tests pay this price...  (Like statically separately build to file first, then run func tests, etc....)